djangocms_versioning/templates/djangocms_versioning/admin/unpublish_confirmation.html

```diff
@@ -0,0 +1,30 @@
+{% extends "admin/base_site.html" %}
+{% load i18n admin_urls static %}
+{% block title %}{% trans "Revert Confirmation" %}{% endblock %}
+
+{% block extrahead %}
+    {{ block.super }}
+    {{ media }}
+    <script type="text/javascript" src="{% static 'admin/js/cancel.js' %}"></script>
+{% endblock %}
+
+{% block breadcrumbs %}{% endblock %}
+{% block bodyclass %}{{ block.super }} app-{{ opts.app_label }} model-{{ opts.model_name }}
+delete-confirmation{% endblock %}
+
+{% block content %}
+    <p>{% trans "Are you sure you want unpublish following version? This will remove page from live website" %} </p>
```

Comment:
Drop "This will remove page from live website". Version doesn't have to refer to Page.